### PR TITLE
Add APIServerURL to response labels

### PR DESCRIPTION
**Pull request by MarekMichali**
**Description**

Changes proposed in this pull request:

- Get APIServerURL from kubeconfig
- Add APIServerURL to response labels
- Add mocks
- Adjust tests

**Related issue(s)**

### CODE REVIEW

File: internal/kubeconfig/builder.go [hunk 1]
```diff
@@ -79,6 +79,25 @@ func (b *Builder) Build(instance *internal.Instance) (string, error) {
...
+		return "", err
+	}
+	err = yaml.Unmarshal(kubeconfigContent, &kubeCfg)
+	if err != nil {
+		return "", fmt.Errorf("while unmarshaling kubeconfig: %w", err)
+	}
+	if err := b.validKubeconfig(kubeCfg); err != nil {
+		return "", fmt.Errorf("while validation kubeconfig fetched by provisioner: %w", err)
+	}
+	return kubeCfg.Clusters[0].Cluster.Server, nil
```
Comment (ralikio) [inline internal/kubeconfig/builder.go:98]: Can you add validation for Clusters number? At least one should be present.

File: internal/kubeconfig/handler.go [hunk 2]
```diff
@@ -75,7 +76,7 @@ func (h *Handler) GetKubeconfig(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	if broker.IsOwnClusterPlan(instance.ServicePlanID) {
+	if h.ownClusterPlanID == instance.ServicePlanID {
```
Comment (ralikio) [inline internal/kubeconfig/handler.go:79]: Why is this line introduced?

File: internal/kubeconfig/handler_test.go [hunk 3]
```diff
@@ -26,6 +25,7 @@ const (
 	instanceID        = "93241a34-8ab5-4f10-978e-eaa6f8ad551c"
 	operationID       = "306f2406-e972-4fae-8edd-50fc50e56817"
 	instanceRuntimeID = "e04813ba-244a-4150-8670-506c37959388"
+	ownClusterPlanID  = "03e3cb66-a4c6-4c6a-b4b0-5d42224debea"
```
Comment (ralikio) [inline internal/kubeconfig/handler_test.go:28]: This plan id is already available by using broker.OwnClusterPlanID.

Comment (MarekMichali) [reply internal/kubeconfig/handler_test.go:28]: If use broker.OwnClusterPlanID we create an import cycle.

File: internal/kubeconfig/handler.go (hunk 2 above)
Comment (MarekMichali) [reply internal/kubeconfig/handler.go:79]: If we leave broker.IsOwnClusterPlan we create an import cycle

File: internal/kubeconfig/builder.go [hunk 4]
```diff
@@ -79,6 +79,25 @@ func (b *Builder) Build(instance *internal.Instance) (string, error) {
 	return b.BuildFromAdminKubeconfig(instance, "")
 }
 
+func (b *Builder) GetServerURL(runtimeID string) (string, error) {
+	if runtimeID == "" {
+		return "", fmt.Errorf("runtimeID must not be empty")
+	}
+	var kubeCfg kubeconfig
+	kubeconfigContent, err := b.kubeconfigProvider.KubeconfigForRuntimeID(runtimeID)
+	if err != nil {
```
[the remark this answers is not in the document]
Comment (MarekMichali) [reply internal/kubeconfig/builder.go:88]: KubeconfigForRuntimeID is also used in other parts of code and we only log this error.

File: internal/kubeconfig/builder.go (hunk 1 above)
Comment (MarekMichali) [reply internal/kubeconfig/builder.go:98]: validKubeconfig does the validation.

File: internal/kubeconfig/builder.go (hunk 4 above)
[the remark this answers is not in the document]
Comment (ralikio) [reply internal/kubeconfig/builder.go:88]: Error is already hidden in `KubeconfigForRuntimeID` method

**Review by ralikio — APPROVED** [review]

**Pull request merged**